src/display/api.js

```diff
@@ -994,16 +1082,224 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
 })();
 
 /**
+ * PDF.js web worker abstraction, it controls instantiation of PDF documents and
+ * WorkerTransport for them.  If creation of a web worker is not possible,
+ * a "fake" worker will be used instead.
+ * @class
+ */
+var PDFWorker = (function PDFWorkerClosure() {
+  var fakeWorkerReady = null;
+  var nextFakeWorkerId = 0;
+
+  // Loads worker code into main thread and setups up the port we will be using
+  // in MessageHandler instances.
+  function setupFakeWorkerGlobal() {
```

Comment:
I actually need to create different instance of the "fakeWorker" for new setupFakeWorkerGlobal request